test.py

```diff
@@ -0,0 +1,5 @@
+from core.feluda import Feluda
```

Comment:
is this file needed? or shall we remove it

Reply:
test.py is just for testing if the packages are running fine. We could remove it and add link to google colab that I mentioned above.

Reply:
yes but there is a `test.py` file in the `package_testing` folder and this one is at the root level, and this file just has the code to init Feluda class, hence wanted to clarity if I missed something or not

Reply:
oh right, I meant to just test if the package are running fine. Sure we can write dedicated tests for it.